wiki/Multiplayer/en.md

```diff
@@ -1,451 +1,419 @@
+
 - Decent online connection.
-- osu!account logged into Bancho (should be done automatically)
-- Have a fair amount of beatmaps in your Songs folder (unless you prefer to download as you go)
-- Knowing [how to transfer beatmaps without leaving the room.][Beatmaps2 wikilink] [Picture example here.][Beatmaps_peppy image]
-- Logical thinking (knowing when to ask about something you don't understand)
+- _osu!account_ logged into _Bancho_ (should be done automatically during ingame login)
+- Have a fair amount of beatmaps in the `Songs` folder (except for personal preference to download on the spot)
+- Knowing [how to transfer beatmaps without leaving the room.][Beatmaps2 wikilink].
+  - Opening the file directly usually works.
 
 ### Ways to enter
+
 From Main menu,
-- Press "**Play**" then "**Multi**"; or
-- Press "**P**" then "**M**". (from Keyboard)
 
-You will be directed to the "**lobby**", the core of the Multi-play.
+- Press `Play` then `Multi`; or
+- Press `P` then `M`. (from Keyboard)
+
+If the connection request with _Bancho_ was successful, _osu!_ will be directed to the **lobby**, the core of the Multi-play.
 
 ### Notes
-- Keep in mind that downloading times can range from a few seconds to 15mins depending on your internet connection.
-- Losing connection to Bancho will send you back to Main Menu until connection is recovered to Bancho. This will force you to be considered "Quit" when playing a match.
-- "Quit" when playing a match.
-- Requires [osu!supporter][osu!supporter wikilink] in cuttingedge build.
+
+Keep in mind that downloading times can range from a few seconds to 15 minutes onwards depending on internet connection.
+
+Losing connection to _Bancho_ will direct _osu!_ back to Main Menu until connection to _Bancho_ has been recovered.
+When playing a match, a "Quit" will be given on the playerbox.
+
+If _not_ in _Stable_ build, the logged _osu!account_ must has an active _[osu!supporter][osu!supporter wikilink]_ to enter Multi.
 
 ## Lobby
+
 ![Example of Multiplayer Lobby][Multi_lobby image]
 
-This is a (waiting) Multiplayer Lobby.
--   Here is a list of things that a player can do in the lobby:-
-	-   Press one of the tabs above to sort out the game style they want.
-	-   Selecting the few options below the tab to sort out the list.
-	-   Creating a new "room" by pressing "New Game"
-	-   Press "Quick Join" to randomly join a room based on the beatmap you have.
-	-   Press one of the "room" to enter that "room" group.
-	-   Talk with other players using the chat console which are opened by default. Please be at your best behavior at all time.
-		-   You will be connected to \#lobby by default.
-	-   Check out the players logged into lobby by looking at the right of the screen.
-	-   Select and playing your jukebox songs at the upper-right of the screen.
+_This is a (waiting) Multiplayer Lobby._
+
+It will show all the available Match Setup in the _Bancho_ server.
```

Comment:
Replaced most instance of "room" (term I invented) with Match Setup (when you enter one) because it was named so.

If nobody in the Match Setup, it will be removed from the Bancho server. A currently-playing Match Setup is still considered available.